kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/MetadataChecksumGeneratorTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.util.UUID;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxy;
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxyBuilder;
+
+class MetadataChecksumGeneratorTest {
+    private static final KafkaProxy PROXY = new KafkaProxyBuilder()
+            .withNewMetadata()
+            .withName("my-proxy")
+            .withUid("my-proxy-uuid")
+            .withGeneration(1L)
+            .endMetadata()
+            .build();
+    // @formatter:on
+
+    @Test
+    void shouldBase64EncodeChecksum() {
+        // Given
+
+        // When
+        String checksum = MetadataChecksumGenerator.checksumFor(PROXY);
+
+        // Then
+        Assertions.assertThat(checksum)
```

Comment:
We've mostly been statically importing assertThat and friends.

Reply:
Yup. Not sure why I missed this one